RULES.md

```diff
@@ -0,0 +1,94 @@
+# Traffic rules
+
+## TLDR
+We need to:
+
+Allow independent groups in the content project to work at their own pace on parts of the project that don’t influence each other.
+Introduce enter and exit criteria for roles with elevated privileges so the hierarchy can self-organize.
```

Comment:
nit: `roles`? Maybe say `individuals and organizations`? IMO there's both individual specific and org-specific stuff below, so perhaps roles might mean something like strictly an organization-nominated maintainers (RH, Canonical, SUSE, Oracle...) and exclude individual contributors unnecessarily... My 2c.

Reply:
Agreed with @cipherboy, it's generally clearer to use specifics, whether that's `individuals and organizations` or what have you, as opposed to `roles`. Rules is too flexible of a term. :)